pkg/common/operator_serviceaccount_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2021 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+// workaround to set the environment variable before the init() function call in operator_serviceaccount.go.
```

Comment:
I didn't know this trick. Sorry I didn't understand your question previously was around testing the init. Probably another way would have been to move the code we call in the init into a function that returns the value and then for UT we could just run the function and then we wouldn't need to work around the init functionality.

Reply:
Removed workaround to set the env var. Test the private function instead as suggested.